src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -243,6 +249,9 @@ export default class ClientMetricReport {
     qpSum: { transform: this.countPerSecond, type: SdkMetric.Type.VIDEO_SENT_QP_SUM },
     frameHeight: { transform: this.identityValue, type: SdkMetric.Type.VIDEO_ENCODE_HEIGHT },
     frameWidth: { transform: this.identityValue, type: SdkMetric.Type.VIDEO_ENCODE_WIDTH },
+    jitter: {
```

Comment:
we already have jitter for audio upstream/downstream and video downstream. So add the same for video upstream.
There is no `type` for this metrics so leave it empty and do not send to Tincan.